tests/unit/slidingPanel/slidingPanelHeader.spec.js

```diff
@@ -66,5 +70,39 @@ describe('SlidingPanel', () => {
       expect(titleEl).toHaveLength(1);
       expect(titleContent).toHaveLength(1);
     });
+
+    it('should render with backButtonLabel and onBackButtonClick', () => {
+      const title = <span className="my-class"> my label </span>;
+      const backButtonClick = jest.fn();
+      const renderTree = mount(
+        <SlidingPanelHeader
+          backButtonLabel="foo"
+          onBackButtonClick={backButtonClick}
+          useDefaultLeftBlock
+        >
+          { title }
+        </SlidingPanelHeader>
+      );
+
+      jest.runAllTimers();
+
+      expect(renderTree).toMatchSnapshot();
+    });
+
+    it('should render with dataAttr and className', () => {
+      const title = <span className="my-class"> my label </span>;
+      const renderTree = mount(
+        <SlidingPanelHeader
+          className="foo"
+          dataAttrs={{ foo: 'bar' }}
+        >
+          { title }
+        </SlidingPanelHeader>
+      );
+
+      jest.runAllTimers();
```

Comment:
Do we need to run all timers here? I noticed you removed similar calls from `slidingPanel.spec.js`, maybe we can remove them from here as well.